src/Webserver.cc

```diff
@@ -121,16 +121,22 @@ bool Webserver::parse_config(const char* file_name) {
 }
 
 RequestHandler* Webserver::get_handler(std::string uri) {
-    size_t last = uri.find_last_of("/");
-    std::string prefix = uri.substr(0, last);
-    std::unordered_map<std::string, RequestHandler*>::const_iterator found = handler_map.find(prefix);
+    std::unordered_map<std::string, RequestHandler*>::const_iterator found = handler_map.find(uri);
```

Comment:
for subdirectories do you mean like path /a StaticHandler, path /a/b StaticHandler?